content/_layouts/docsproject.html.haml

```diff
@@ -0,0 +1,47 @@
+---
+layout: project
+project: docs
```

Comment:
Could we somehow merge it with `gsocprojectidea` layouts? IMHO DRY

Reply:
Yes, once I've seen how the layout works (some changing concepts, like "Writer" or "Author" rather than "Student", and some patterns that I'm not yet sure how to apply, like year)

Reply:
You can parameterize it or change to a default "mentee"